src/list/manage/components/edit-item-details.js

```diff
@@ -69,9 +70,26 @@ export default class EditItemDetails extends React.Component {
               e.preventDefault();
               onSave(saveState);
             }}>
-        <Localized id={`item-details-heading-${newItem ? "new" : "edit"}`}>
-          <h1>eDIt iTEm</h1>
-        </Localized>
+        <header>
+          {newItem ? (
+            <Localized id={`item-details-heading-new`}>
+              <h1>nEw iTEm</h1>
+            </Localized>
+          ) : (
+            <React.Fragment>
```

Comment:
TIL about `React.Fragment` 🌈 🌟

Reply:
Yeah, I think you can also use `<>` and `</>` but I like explicitly naming the `React.Fragment` component.